src/config/environment.js

```diff
@@ -11,5 +11,6 @@ module.exports = {
   SERVICE_AVAILABLE_DATE: getVCAPServicesVariable('SERVICE_AVAILABLE_DATE'),
   OVERVIEW_URL: '/',
   APP_VERSION: process.env.APP_VERSION || 'unknown',
-  NOTIFY_API_KEY: getVCAPServiceNotifyVariable('NOTIFY_API_KEY', process.env.NOTIFY_API_KEY)
+  NOTIFY_API_KEY: getVCAPServiceNotifyVariable('NOTIFY_API_KEY', process.env.NOTIFY_API_KEY),
+  OS_PLACES_API_KEY: getVCAPServiceOSPlacesVariable('OS_PLACES_API_KEY', process.env.OS_PLACES_API_KEY)
```

Comment:
Can this variable be added to `sample.env`?

Reply:
Added.